### ISSUE TITLE
New submissions for Thu, 26 Oct 23

### ISSUE THREAD

**Issue by olozhika** [olozhika/ArXivDaily_StarFormation#133]
# New submissions for Thu, 26 Oct 23
Auto update Star Formation & Molecular Cloud papers at about 2:30am UTC (10:30am Beijing time) every weekday.

阅读 `Usage.md`了解如何使用此repo实现个性化的Arxiv论文推送

See `Usage.md` for instructions on how to personalize the repo. 

Keyword list: ['star formation', 'molecular cloud', 'interstellar medium', 'cloud', 'clump', 'core', 'filament', 'atomic gas', 'N-PDF']

Excluded: ['galaxies', 'galaxy cluster', 'AGN']

#### Physical Properties of 5,000 Cool LMC Supergiants with Gaia XP Spectra:  A Detailed Portrait of the Upper HR Diagram Hints at Missing Supernova  Progenitors
 - **Authors:** Trevor Z. Dorn-Wallenstein, Kathryn F. Neugent, Emily M. Levesque
 - **Subjects:** Solar and Stellar Astrophysics (astro-ph.SR)
 - **Arxiv link:** https://arxiv.org/abs/2310.16088
 - **Pdf link:** https://arxiv.org/pdf/2310.16088
 - **Abstract**
 Characterizing the physical properties of cool supergiants allows us to probe the final stages of a massive star's evolution before it undergoes core collapse. Despite their importance, the fundamental properties for these stars -- $T_{\rm eff}$ and $\log L/L_\odot$ -- are only known for a limited number of objects. The third data release of the Gaia mission contains precise photometry and low-resolution spectroscopy of hundreds of cool supergiants in the LMC with well-constrained properties. Using these data, we train a simple and easily-interpretable machine learning model to regress effective temperatures and luminosities with high accuracy and precision comparable to the training data. We then apply our model to 5000 cool supergiants, many of which have no previously-published $T_{\rm eff}$ or $L$ estimates. The resulting Hertzprung-Russell diagram is well-populated, allowing us to study the distribution of cool supergiants in great detail. Examining the luminosity functions of our sample, we find a notable flattening in the luminosity function of yellow supergiants above $\log L/L_\odot=5$, and a corresponding steepening of the red supergiant luminosity function. We place this finding in context with previous results, and present its implications for the infamous red supergiant problem.
#### SN 2022jox: An extraordinarily ordinary Type II SN with Flash  Spectroscopy
 - **Authors:** Jennifer E. Andrews, Jeniveve Pearson, Griffin Hosseinzadeh, K. Azalee Bostroem, Yize Dong, Manisha Shrestha, Jacob E. Jencson, David J. Sand, S. Valenti, Emily Hoang, Daryl Janzen, M.J. Lundquist, Nicolas Meza, Samuel Wyatt, Saurabh W. Jha, Chris Simpson, Joseph Farah, Estefania Padilla Gonzalez, D. Andrew Howell, Curtis McCully, Megan Newsome, Craig Pellegrino, Giacomo Terreran
 - **Subjects:** High Energy Astrophysical Phenomena (astro-ph.HE); Solar and Stellar Astrophysics (astro-ph.SR)
 - **Arxiv link:** https://arxiv.org/abs/2310.16092
 - **Pdf link:** https://arxiv.org/pdf/2310.16092
 - **Abstract**
 We present high cadence optical and ultraviolet observations of the Type II supernova (SN), SN 2022jox which exhibits early spectroscopic high ionization flash features of \ion{H}{1}, \ion{He}{2}, \ion{C}{4}, and \ion{N}{4} that disappear within the first few days after explosion. SN 2022jox was discovered by the Distance Less than 40 Mpc (DLT40) survey $\sim$0.75 days after explosion with followup spectra and UV photometry obtained within minutes of discovery. The SN reached a peak brightness of M$_V \sim$ $-$17.3 mag, and has an estimated $^{56}$Ni mass of 0.04 M$_{\odot}$, typical values for normal Type II SNe. The modeling of the early lightcurve and the strong flash signatures present in the optical spectra indicate interaction with circumstellar material (CSM) created from a progenitor with a mass loss rate of $\dot{M} \sim 10^{-3}-10^{-2}\ M_\odot\ \mathrm{yr}^{-1}$. There may also be some indication of late-time CSM interaction in the form of an emission line blueward of H$\alpha$ seen in spectra around 200 days. The mass-loss rate is much higher than the values typically associated with quiescent mass loss from red supergiants, the known progenitors of Type II SNe, but is comparable to inferred values from similar core collapse SNe with flash features, suggesting an eruptive event or a superwind in the progenitor in the months or years before explosion.
#### Dwindling Surface Cooling of a Rotating Jovian Planet Leads to a  Convection Zone that Grows to a Finite Depth
 - **Authors:** Bradley W. Hindman, J.R. Fuentes
 - **Subjects:** Earth and Planetary Astrophysics (astro-ph.EP)
 - **Arxiv link:** https://arxiv.org/abs/2310.16124
 - **Pdf link:** https://arxiv.org/pdf/2310.16124
 - **Abstract**
 Recent measurements of Jupiter's gravitational field (by Juno) and seismology of Saturn's rings (by Cassini) strongly suggest that both planets have a stably-stratified core that still possesses a primordial gradient in the concentration of heavy elements. The existence of such a "diffusely" stratified core has been a surprise as it was long expected that the Jovian planets should be fully convective and hence fully mixed. A vigorous zone of convection, driven by surface cooling, forms at the surface and deepens through entrainment of fluid from underneath. In fact, it was believed that this convection zone should grow so rapidly that the entire planet would be consumed in less than a million years. Here we suggest that two processes, acting in concert, present a solution to this puzzle. All of the giant planets are rapidly rotating and have a cooling rate that declines with time. Both of these effects reduce the rate of fluid entrainment into the convection zone. Through the use of an analytic prescription of entrainment in giant planets, we demonstrate that these two effects, rotation and dwindling surface cooling, result in a convection zone which initially grows but eventually stalls. The depth to which the convective interface asymptotes depends on the rotation rate and on the stratification of the stable interior. Conversely, in a nonrotating planet, or in a planet that maintains a higher level of cooling than current models suggest, the convection zone deepens forever, eventually spanning the entire planet.
#### Simultaneously probing the sound speed and equation of state of the  early Universe with pulsar timing arrays
 - **Authors:** Lang Liu, You Wu, Zu-Cheng Chen
 - **Subjects:** Cosmology and Nongalactic Astrophysics (astro-ph.CO); General Relativity and Quantum Cosmology (gr-qc)
 - **Arxiv link:** https://arxiv.org/abs/2310.16500
 - **Pdf link:** https://arxiv.org/pdf/2310.16500
 - **Abstract**
 Recently, several major pulsar timing array (PTA) collaborations have assembled strong evidence for the existence of a gravitational-wave background at frequencies around the nanohertz regime. Assuming that the PTA signal is attributed to scalar-induced gravitational waves, we jointly employ the PTA data from the NANOGrav 15-year data set, PPTA DR3, and EPTA DR2 to probe the conditions of the early Universe. Specifically, we explore the equation of state parameter ($w$), the sound speed ($c_s$), and the reheating temperature ($T_\mathrm{rh}$), finding $w=0.60^{+0.32}_{-0.39}$, $c_s\gtrsim 0.09$, and $T_\mathrm{rh}\lesssim 0.2\,\mathrm{GeV}$ for a lognormal power spectrum of the curvature perturbation. Furthermore, we compute Bayes factors to compare different models against the radiation domination model ($c_s^2 = w = 1/3$), effectively excluding the pressure-less fluid domination model. Our study underscores the significance of scalar-induced gravitational waves as a powerful tool to explore the nature of the early Universe.
#### NIKA2 observations of dust grain evolution from star-forming filament to  T-Tauri disk: Preliminary results from NIKA2 observations of the Taurus  B211/B213 filament
 - **Authors:** Q. Nguyen-Luong, R. Adam, P. Ade, H. Ajeddig, P. André, E. Artis, H. Aussel, A. Beelen, A. Benoît, S. Berta, L. Bing, O. Bourrion, M. Calvo, A. Catalano, M. De Petris, F.-X. Désert, S. Doyle, E. F. C. Driessen, G. Ejlali, A. Gomez, J. Goupy, C. Hanser, S. Katsioli, F. Kéruzoré, C. Kramer, B. Ladjelate, G. Lagache, S. Leclercq, J.-F. Lestrade, J. F. Macías-Pérez, S. C. Madden, A. Maury, P. Mauskopf, F. Mayet, A. Monfardini, A. Moyer-Anin, M. Muñoz-Echeverría, L. Perotto, G. Pisano, N. Ponthieu, V. Revéret, A. J. Rigby, A. Ritacco, C. Romero, H. Roussel, F. Ruppin, K. Schuster, A. Sievers, C. Tucker, R. Zylka, A. Bacmann, A. Duong-Tuan, N. Peretto, A. Rigby
 - **Subjects:** Solar and Stellar Astrophysics (astro-ph.SR); Astrophysics of Galaxies (astro-ph.GA)
 - **Arxiv link:** https://arxiv.org/abs/2310.16526
 - **Pdf link:** https://arxiv.org/pdf/2310.16526
 - **Abstract**
 To understand the evolution of dust properties in molecular clouds in the course of the star formation process, we constrain the changes in the dust emissivity index from star-forming filaments to prestellar and protostellar cores to T Tauri stars. Using the NIKA2 continuum camera on the IRAM 30~m telescope, we observed the Taurus B211/B213 filament at 1.2\,mm and 2\,mm with unprecedented sensitivity and used the resulting maps to derive the dust emissivity index $\beta$. Our sample of 105 objects detected in the $\beta$ map of the B211/B213 filament indicates that, overall, $\beta$ decreases from filament and prestellar cores ($\beta \sim 2\pm0.5$) to protostellar cores ($\beta \sim 1.2 \pm 0.2$) to T-Tauri protoplanetary disk ($\beta < 1$). The averaged dust emissivity index $\beta$ across the B211/B213 filament exhibits a flat ($\beta \sim 2\pm0.3$) profile. This may imply that dust grain sizes are rather homogeneous in the filament, start to grow significantly in size only after the onset of the gravitational contraction/collapse of prestellar cores to protostars, reaching big sizes in T Tauri protoplanetary disks. This evolution from the parent filament to T-Tauri disks happens on a timescale of about 1-2~Myr.
#### Study of the fastest classical nova, V1674 Her: Photoionization and  Morpho-kinemetic model analysis
 - **Authors:** Gesesew R. Habtie, Ramkrishna Das, Ruchi Pandey, N. M. Ashok, Pavol A. Dubovsky
 - **Subjects:** Solar and Stellar Astrophysics (astro-ph.SR)
 - **Arxiv link:** https://arxiv.org/abs/2310.16619
 - **Pdf link:** https://arxiv.org/pdf/2310.16619
 - **Abstract**
 We present the results of the investigation of the nova V1674 Her (2021), recognised as the swiftest classical nova, with $t_2 \sim 0.90$ days. The distance to the nova is estimated to be 4.97 kpc. The mass and radius of the WD are calculated to be $\sim~1.36~M_\odot$ and $\sim 0.15~R_\oplus$, respectively. Over the course of one month following the outburst, V1674 Her traversed distinct phases -- pre-maxima, early decline, nebular, and coronal -- displaying a remarkably swift transformation. The nebular lines emerged on day 10.00, making it the classical nova with the earliest observed commencement to date. We modelled the observed optical spectrum using the photoionization code \textsc{cloudy}. From the best-fitting model we deduced different physical and chemical parameters associated withe the system. The temperature and luminosity of the central ionizing sources are found in the range of $1.99 - 2.34~\times 10^5$ K and $1.26 - 3.16~ \times 10^{38}$ \ergs, respectively. Elements such as He, O, N, and Ne are found to be overabundant compared to solar abundance in both the nebular and coronal phases. According to the model, Fe II abundance diminishes while Ne abundance increases, potentially elucidating the rare hybrid transition between Fe and He/N nova classes. The ejected mass across all epochs spanned from $3.42 - 7.04~ \times 10^{-5}~M_\odot$. Morpho-kinematic modelling utilising \textsc{shape} revealed that the nova V1674 Her possesses a bipolar structure with an equatorial ring at the centre and an inclination angle of i = 67$\pm$ 1.5$^{\circ}$.
#### The Simplest Inflationary Potentials
 - **Authors:** Tomás Sousa, Deaglan J. Bartlett, Harry Desmond, Pedro G. Ferreira
 - **Subjects:** Cosmology and Nongalactic Astrophysics (astro-ph.CO); Machine Learning (cs.LG); General Relativity and Quantum Cosmology (gr-qc); High Energy Physics - Phenomenology (hep-ph); High Energy Physics - Theory (hep-th)
 - **Arxiv link:** https://arxiv.org/abs/2310.16786
 - **Pdf link:** https://arxiv.org/pdf/2310.16786
 - **Abstract**
 Inflation is a highly favoured theory for the early Universe. It is compatible with current observations of the cosmic microwave background and large scale structure and is a driver in the quest to detect primordial gravitational waves. It is also, given the current quality of the data, highly under-determined with a large number of candidate implementations. We use a new method in symbolic regression to generate all possible simple scalar field potentials for one of two possible basis sets of operators. Treating these as single-field, slow-roll inflationary models we then score them with an information-theoretic metric ("minimum description length") that quantifies their efficiency in compressing the information in the Planck data. We explore two possible priors on the parameter space of potentials, one related to the functions' structural complexity and one that uses a Katz back-off language model to prefer functions that may be theoretically motivated. This enables us to identify the inflaton potentials that optimally balance simplicity with accuracy at explaining the Planck data, which may subsequently find theoretical motivation. Our exploratory study opens the door to extraction of fundamental physics directly from data, and may be augmented with more refined theoretical priors in the quest for a complete understanding of the early Universe.
#### Uncovering a new group of T Tauri stars in the Taurus-Auriga molecular  complex from Gaia and GALEX data
 - **Authors:** Ana Inés Gómez de Castro, Raúl de la Fuente Marcos, Ada Canet, Leire Beitia-Antero, Javier Yañez-Gestoso, Juan Carlos Vallejo
 - **Subjects:** Solar and Stellar Astrophysics (astro-ph.SR); Instrumentation and Methods for Astrophysics (astro-ph.IM)
 - **Arxiv link:** https://arxiv.org/abs/2310.16820
 - **Pdf link:** https://arxiv.org/pdf/2310.16820
 - **Abstract**
 In this work, we examine the list of 63 candidates to T Tauri star (TTS) in the TAMC identified by their ultraviolet (UV) and infrared colours (IR) measured from data obtained by the Galaxy Evolution Explorer all sky survey (GALEX-AIS) and the Two Microns All Sky Survey (2MASS), respectively. The objective of this work is twofold: evaluate whether they are pre-main sequence (PMS) stars and evaluate the goodness of the UV-IR colour-colour diagram to detect PMS stars in wide-fields. The astrometric properties of these sources have been retrieved from the Gaia DR3 catalogue and used to evaluate their membership probability. Several classification algorithms have been tested to search for the kinematical groups but the final classification has been made with k-means++ algorithms. Membership probability has been evaluated by applying Logistic Regression. In addition, spectroscopic information available in the archive of the Large Sky Area Multi Object Fiber Spectroscopic Telescope has been used to ascertain their PMS nature when available. About 20% of the candidates share the kinematics of the TAMC members. Among them, HD 281691 is a G8-type field star located in front of the cloud and HO Aur is likely a halo star given the very low metallicity provided by Gaia. The rest are three known PMS stars (HD 30171, V600 Aur and J04590305+3003004), two previously unknown accreting M-type stars (J04510713+1708468 and J05240794+2542438) and, five additional sources, which are very likely PMS stars. Most of these new sources are concentrated at low galactic latitudes over the Auriga-Perseus region.
#### Deep machine learning for meteor monitoring: advances with transfer  learning and gradient-weighted class activation mapping
 - **Authors:** Eloy Peña-Asensio, Josep M. Trigo-Rodríguez, Pau Grèbol-Tomàs, David Regordosa-Avellana, Albert Rimola
 - **Subjects:** Earth and Planetary Astrophysics (astro-ph.EP); Instrumentation and Methods for Astrophysics (astro-ph.IM); Machine Learning (cs.LG); Image and Video Processing (eess.IV)
 - **Arxiv link:** https://arxiv.org/abs/2310.16826
 - **Pdf link:** https://arxiv.org/pdf/2310.16826
 - **Abstract**
 In recent decades, the use of optical detection systems for meteor studies has increased dramatically, resulting in huge amounts of data being analyzed. Automated meteor detection tools are essential for studying the continuous meteoroid incoming flux, recovering fresh meteorites, and achieving a better understanding of our Solar System. Concerning meteor detection, distinguishing false positives between meteor and non-meteor images has traditionally been performed by hand, which is significantly time-consuming. To address this issue, we developed a fully automated pipeline that uses Convolutional Neural Networks (CNNs) to classify candidate meteor detections. Our new method is able to detect meteors even in images that contain static elements such as clouds, the Moon, and buildings. To accurately locate the meteor within each frame, we employ the Gradient-weighted Class Activation Mapping (Grad-CAM) technique. This method facilitates the identification of the region of interest by multiplying the activations from the last convolutional layer with the average of the gradients across the feature map of that layer. By combining these findings with the activation map derived from the first convolutional layer, we effectively pinpoint the most probable pixel location of the meteor. We trained and evaluated our model on a large dataset collected by the Spanish Meteor Network (SPMN) and achieved a precision of 98\%. Our new methodology presented here has the potential to reduce the workload of meteor scientists and station operators and improve the accuracy of meteor tracking and classification.

by olozhika (Xing Yuchen). 

2023-10-26
